kmean_cluster: 10
imgtest_folder: "/home/jsaraydaryan/ros_robotcupathome_ws/src/people_management/ros_color_detection/ros_color_detection_node/data"
is_process_displayed: True